snyk_threadfix/main.py

```diff
@@ -251,16 +255,27 @@ def main(args):
 
     all_threadfix_findings = []
 
-    for p_id in project_ids:
-        threadfix_findings = create_threadfix_findings_data(args.org_id, p_id)
-        all_threadfix_findings.extend(threadfix_findings)
+    try:
+        for p_id in project_ids:
+            threadfix_findings = create_threadfix_findings_data(args.org_id, p_id)
+            all_threadfix_findings.extend(threadfix_findings)
 
-    threadfix_json_obj['findings'] = all_threadfix_findings
+        threadfix_json_obj['findings'] = all_threadfix_findings
 
-    if args.output:
-        write_to_threadfix_file(args.output, threadfix_json_obj)
-    else:
-        write_output_to_stdout(threadfix_json_obj)
+        if args.output:
+            write_to_threadfix_file(args.output, threadfix_json_obj)
+        else:
+            write_output_to_stdout(threadfix_json_obj)
+
+    except snyk.errors.SnykOrganizationNotFoundError:
+        log_error('Error resolving org in Snyk. This is probably because your `--org-id` parameter value is invalid.')
```

Comment:
@garethr - this logs to stderr but do you think that in this case (where we have a fatal error) that we should send it to stdout? Either way, I'm logging it regardless of whether a user uses the `--debug` flag.